sdk/containerinstance/Microsoft.Azure.Management.ContainerInstance/src/Generated/ContainersOperationsExtensions.cs

```diff
@@ -72,12 +76,16 @@ public static Logs ListLogs(this IContainersOperations operations, string resour
             /// The number of lines to show from the tail of the container instance log. If
             /// not provided, all available logs are shown up to 4mb.
             /// </param>
+            /// <param name='timestamps'>
+            /// If true, adds a timestamp at the beginning of every line of log output. If
+            /// not provided, defaults to false.
+            /// </param>
             /// <param name='cancellationToken'>
             /// The cancellation token.
             /// </param>
-            public static async Task<Logs> ListLogsAsync(this IContainersOperations operations, string resourceGroupName, string containerGroupName, string containerName, int? tail = default(int?), CancellationToken cancellationToken = default(CancellationToken))
+            public static async Task<Logs> ListLogsAsync(this IContainersOperations operations, string resourceGroupName, string containerGroupName, string containerName, int? tail = default(int?), bool? timestamps = default(bool?), CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Per previous comment, this will also require a customization

Reply:
Added overload in a custom partial interface